index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mi CV</title>
+</head>
+<body>
+    <div class="contenedor">

```

Comment:
Debemos programar en inglés

```suggestion
    <div class="container">

```